scripts/metrics/pytest_metrics_xml_to_csv.py

```diff
@@ -0,0 +1,163 @@
+"""
+Copyright (c) Contributors to the Open 3D Engine Project.
+For complete copyright and license terms please see the LICENSE at the root of this distribution.
+
+SPDX-License-Identifier: Apache-2.0 OR MIT
+
+Scrapes metrics from Pytest xml files and creates csv formatted files.
+"""
+import os.path
+import sys
+import csv
+import argparse
+import xml.etree.ElementTree as xmlElementTree
```

Comment:
I think adding a lambda would overcomplicate the pipeline because the xml files are cleaned up after every pipeline run. We would have to persist the files, trigger a lambda to execute the script, then clean up the old workspace.

Reply:
Mostly brought this up as https://github.com/o3de/sig-build/issues/77 mentions moving operations to GitHub actions, which are quite similar to AWS Lambda. But I think we can defer that discussion outside of this PR. Becomes a question of whether persisting the XML to another stage is cheaper/easier.